pkgs/applications/version-management/git-and-tools/legit/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "legit";
+  version = "1.1.0";
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "0kbvczs0rnzazxqx8nc10rp6x9aqgzf318m68nyv1japsq4kig95";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    click
+    clint
+    crayons
+    GitPython
+    six
+  ];
+
+  # Unpin dependency versions.
+  postPatch = ''
+    sed -i 's/==.*//g' reqs.txt
+  '';
+
+  # Prevent tests from trying to create configuration in /homeless-shelter.
+  preCheck = ''
+    export HOME=$PWD/test-home
+  '';
+
+  meta = with lib; {
+    homepage = https://github.com/frostming/legit;
```

Comment:
```suggestion
    homepage = "https://github.com/frostming/legit";
```

Reply:
Is there a good reason to start quoting urls? This doesn't seem typical.

Reply:
Yes, https://github.com/NixOS/rfcs/pull/45